packages/wrangler/src/index.tsx

```diff
@@ -558,6 +558,7 @@ export async function main(argv: string[]): Promise<void> {
           }}
         />
       );
+      await waitUntilExit();
```

Comment:
https://github.com/vadimdemedes/ink#error
So from what I can tell this now exits with an error using the Ink API, what is the `waitUntilExit` handling? Is it unmounting after the `exit()` is called in the `useEffect`?

Reply:
This will block the exit of the `Dev` "app" until we explicitly exit via a call to `useApp().exit()`.
If the `exit()` call passes an error object then the promise here will reject. Otherwise it will resolve.